server/src/server.js

```diff
@@ -12,7 +12,7 @@ var wss = new WebSocketServer({ port: port });
 
 var players = [];
 var player_count = 0;
-var accepted_server_url = '';
+var accepted_server = [];
```

Comment:
`accepted_server` is used as an object in the following code, so I think it should be initialized as an object instead of a string.
But I think it doesn't affect the behavior no matter what it is initialized as. Because it will be set to `player.server` when the first player connect to the server.